arangod/Aql/ClusterBlocks.cpp

```diff
@@ -1181,12 +1188,13 @@ SortingGatherBlock::SortingGatherBlock(ExecutionEngine& engine, GatherNode const
   TRI_ASSERT(!en.elements().empty());
 
   switch (en.sortMode()) {
-    case GatherNode::SortMode::Heap:
-      _strategy = std::make_unique<HeapSorting>(_trx, _gatherBlockBuffer, _sortRegisters);
-      break;
     case GatherNode::SortMode::MinElement:
       _strategy = std::make_unique<MinElementSorting>(_trx, _gatherBlockBuffer, _sortRegisters);
       break;
+    case GatherNode::SortMode::Heap:
+    case GatherNode::SortMode::Default: // use heap by default
```

Comment:
There was no "default" strategy before, I've added `Default` value since views don't know how many db servers will be involved into distribution at planning phase. So that didn't change behavior.